src/test/scala/de/zalando/beard/parser/BeardTemplateParserSpec.scala

```diff
@@ -167,6 +167,16 @@ class BeardTemplateParserSpec extends FunSpec with Matchers {
     }
   }
 
+  describe("when parsing a string that contains a comment") {
```

Comment:
could you please add a test where the comment is surrounded by other things, to be sure that the other things are maintained?

    Hello {{# comment #}} world

Also a test with `Hello {{# text }} {{  #}} world` to be sure that `{{` are not taken into account